python: can't open file 'task2_convnet.py': [Errno 2] No such file or directory
